Making Sense/Models/Program.cs

```diff
@@ -51,7 +52,27 @@ static void Main(string[] args)
             Console.WriteLine("El auto actualizado: ");
             Console.WriteLine(Cars.Update(Corsa).ToString());
             Cars.Delete(1);
-
+            Customer cliente = new Customer
+            {
+                DNI = 36617481,
+                Name = "Facundo",
+                Surname = "Rueda",
+                Telephone = "2235378667",
+                Adress = "Friuli 797",
+                City =  "Mar del Plata",
+                State = "Buenos Aires",
+                ZipCode = 7600,
+                LastModificationDate = "12/09/2021"
+            };
+            Console.WriteLine(cliente.ToString());
+            Rental Rental = new Rental
```

Comment:
Un detalle de convenciones:
```suggestion
            var Rental = new Rental
```